src/mlpack/methods/ann/activation_functions/multi_quadratic_function.hpp

```diff
@@ -0,0 +1,81 @@
+/**
+ * @file methods/ann/activation_functions/multi_quadratic_function.hpp
+ * @author Himanshu Pathak
+ *
+ * Definition and implementation of multi quadratic function.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef MLPACK_METHODS_ANN_ACTIVATION_FUNCTIONS_MULTIQUAD_FUNCTION_HPP
+#define MLPACK_METHODS_ANN_ACTIVATION_FUNCTIONS_MULTIQUAD_FUNCTION_HPP
+
+#include <mlpack/prereqs.hpp>
+
+namespace mlpack {
+namespace ann /** Artificial Neural Network. */ {
+
+/**
+ * The Multi Quadratic function, defined by
+ *
+ * @f{eqnarray*}{
+ * f(x) = √(1 + x^2) \\
```

Comment:
I'm not sure doxygen can render that in a latex environment, so maybe we should use `\sqrt` here.

Reply:
Thanks for reviewing.